lib/components/Spotlight/Spotlight.tsx

```diff
@@ -33,6 +33,17 @@ export default function Spotlight<T extends object>({
   showSeeMoreButton,
   onSeeMoreClick,
   seeMoreHref,
+  className,
+  contentClassName,
+  iconClassName,
+  inputClassName,
+  buttonClassName,
+  noResultsClassName,
+  recentSearchClassName,
+  buttonLabel,
```

Comment:
Coloca os valores padrões aqui, assim:

```suggestion
  buttonLabel = "See more",
```